pages/common/fastmod.md

```diff
@@ -0,0 +1,29 @@
+# fastmod
+
+> A fast partial replacement for the codemod tool, replace and replace all in the whole codebase.
+> Regexes are matched by Rust regex crate.
+> More information: <https://github.com/facebookincubator/fastmod>.
+
+- Replace regex pattern in all files of the current directory, ignoring files on .ignore and .gitignore:
+
+`fastmod {{regex_pattern}} {{replacement}}`
+
+- Case-insensitively replace regex pattern in specific files even if ignored, or directory ignoring files on .ignore and .gitignore:
```

Comment:
The new description doesn't clarify that if you specify a file it will be included even if it is ignore in `.ignore` or `.gitignore`

Reply:
It says `ignoring files on` :thinking:  maybe this is better:
```suggestion
- Replace a regex pattern in case-insensitive mode in specific files or directories, ignoring .ignore and .gitignore:
```

Reply:
> Replace a regex pattern in a case-insensitive mode in specific files or directories:

I think this is a better description because all these examples files are ignored using .ignore or .gitignore which is emphasized in the first example.